stdlib/Random/src/misc.jl

```diff
@@ -71,13 +71,14 @@ let b = UInt8['0':'9';'A':'Z';'a':'z']
     global randstring
 
     function randstring(r::AbstractRNG, chars=b, n::Integer=8)
+        _n = convert(Int, n)
```

Comment:
How about this:

```suggestion
        n = convert(Int, n)::Int
```

Then the rest of the diff can be dropped.